docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.4'
+
+services:
+  data-hub-api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: runtime
```

Comment:
we currently aren't defining a target in the dockerfile, does this still build fine via docker compose?